docs/DeveloperGuide.md

```diff
@@ -435,6 +435,52 @@ testers are expected to do more *exploratory* testing.
     1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
+### Creating a pool
+
+1. Creating a pool from using the list of all passengers being shown.
+
+    1. Prerequisites: List all passengers using the `list` command.
+       Multiple passengers in the list, with the first two passengers having the same the trip day of `MONDAY` and trip time of `1930`.
```

Comment:
Maybe you could consider suggesting to the user that they could consider editing the passengers to Monday 1930 if necessary